plugins/kiali/DEVELOPMENT.md

```diff
@@ -121,7 +123,7 @@ async function main() {
   apiRouter.use('/kiali', await kiali(kialiEnv));
 ```
 
-6. Configure you `app-config.yaml` with kiali configuration
+6. Configure your `app-config.yaml` with kiali configuration
```

Comment:
Shouldn't this be done on `app-config.local.yaml` since this is specifically about development time.